rumqttc/examples/async_manual_acks.rs

```diff
@@ -50,16 +50,14 @@ async fn main() -> Result<(), Box<dyn Error>> {
         // previously published messages should be republished after reconnection.
         let event = eventloop.poll().await;
         println!("{:?}", event);
-        match event {
-            Ok(Event::Incoming(Incoming::Publish(publish))) => {
-                // this time we will ack incoming publishes.
-                // Its important not to block eventloop as this can cause deadlock.
-                let c = client.clone();
-                tokio::spawn(async move {
-                    c.ack(&publish).await.unwrap();
-                });
-            }
-            _ => {}
+
+        if let Ok(Event::Incoming(Incoming::Publish(publish))) = event {
```

Comment:
Clippy suggestions, implemented to ensure CI doesn't warn, i.e. cause CI to fail